cbc.py

```diff
@@ -0,0 +1,26 @@
+from Cryptodome.Cipher import AES
```

Comment:
@talvasconcelos can we remove this file or do you plan to use it somewhere?

Reply:
it can be removed, what was needed it's on helpers.py